flake.nix

```diff
@@ -21,5 +21,13 @@
         formatter = pkgs.nixpkgs-fmt;
 
         devShells.default = import ./nix/shell.nix { inherit pkgs inputs; };
-      });
+      }) 
+    // rec {
+      overlays = {
+        default = overlays.yazi;
+        yazi = final: prev: {
+          yazi = final.callPackage ./nix/yazi.nix { };
+        };
+      };
+    };
```

Comment:
```suggestion
    // {
      overlays = rec {
        default = yazi;
        yazi = final: prev: {
          yazi = self.packages."${final.system}".yazi;
        };
      };
    };
```

Reply:
Thanks, didn't know that you can obtain the system information this way! I thought the only way to obtain it is from `builtins.currentSystem` which is impure and not allowed in flakes.